ops/services/app_functions/report_stream_batched_publisher/infra/_var.tf

```diff
@@ -35,6 +35,11 @@ variable "test_event_queue_name" {
   default = "test-event-publishing"
 }
 
+variable "file_error_queue_name" {
+  type    = string
+  default = "test-event-file-error"
```

Comment:
proposing to change the default value to `test-event-publishing-error` since its a more generic term, and tbh, the word `file-error` is a bit misleading since its a 400 error